libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -264,58 +246,7 @@ - (void)testMultipleAccounts {
         
         [self deleteUserAndVerify:userAccount userDir:location];
     }
-}
-
-- (void)testTemporaryAccount {
-    SFUserAccount *tempAccount1 = [self.uam userAccountForUserIdentity:self.uam.temporaryUserIdentity];
-    XCTAssertNil(tempAccount1, @"Temp account should not be defined here.");
-    SFUserAccount *tempAccount2 = self.uam.temporaryUser;
-    XCTAssertNotNil(tempAccount2, @"Temp account should be created through the temporaryUser property.");
-    tempAccount1 = [self.uam userAccountForUserIdentity:self.uam.temporaryUserIdentity];
-    XCTAssertEqual(tempAccount1, tempAccount2, @"Temp account references should be equal.");
-}
-
-- (void)testAnonymousUser {
-    NSArray *accounts = [self createAndVerifyUserAccounts:1];
-    SFUserAccount *origUser = accounts[0];
-    XCTAssertNil(self.uam.anonymousUser, @"Anonymous user shouldn't exist yet");
-    XCTAssertNil(self.uam.currentUser, @"Current user shouldn't be set yet");
-    [self.uam setupAnonymousUser:YES autocreateAnonymousUser:NO];
-    XCTAssertNil(self.uam.anonymousUser, @"Anonymous user shouldn't exist yet");
-    [self.uam setupAnonymousUser:YES autocreateAnonymousUser:YES];
-    XCTAssertNotNil(self.uam.anonymousUser, @"Anonymous user should exist now");
-    XCTAssertTrue(self.uam.currentUser == self.uam.anonymousUser, @"Current user should be the anonymous user");
-    self.uam.currentUser = origUser;
-    XCTAssertTrue(self.uam.anonymousUser.isAnonymousUser, @"Anonymous user should be who he is");
-    XCTAssertFalse(self.uam.anonymousUser.isTemporaryUser, @"Anonymous user shouldn't be temporary");
-    XCTAssertFalse(self.uam.currentUser.isAnonymousUser, @"Current user shouldn't be anonymous");
-    XCTAssertFalse(self.uam.currentUser.isTemporaryUser, @"Current user shouldn't be temporary");
-}
-
-- (void)testAnonymousUserFromPreviousVersion {
-    XCTAssertNil(self.uam.anonymousUser, @"Anonymous user shouldn't exist yet");
-    XCTAssertNil(self.uam.currentUser, @"Current user shouldn't be set yet");
-    NSArray *accounts = [self createAndVerifyUserAccounts:1];
-    SFUserAccount *origUser = accounts[0];
-    self.uam.currentUser = origUser;
-    [self.uam setupAnonymousUser:YES autocreateAnonymousUser:NO];
-    XCTAssertNil(self.uam.anonymousUser, @"Anonymous user shouldn't exist yet");
-    [self.uam setupAnonymousUser:YES autocreateAnonymousUser:YES];
-    XCTAssertNotNil(self.uam.anonymousUser, @"Anonymous user should exist now");
-    XCTAssertTrue(self.uam.currentUser == origUser, @"Current user should be the anonymous user");
-}
-
-- (void)testSwitchToNewUser {
```

Comment:
I don't see why this test goes away, rather than changing the evaluation at the end that `currentUser` should be `nil`.  This is still a valid test.